extension/src/config/config.js

```diff
@@ -27,6 +27,9 @@ function getCtpConfig(ctpProjectKey) {
       ctpConfig.apiUrl || 'https://api.europe-west1.gcp.commercetools.com',
     authUrl:
       ctpConfig.authUrl || 'https://auth.europe-west1.gcp.commercetools.com',
+    isAuthEnabled: ctpConfig.isAuthEnabled || false,
+    username: ctpConfig.username || '',
```

Comment:
I think here you don't need a default value for username and password. Will it work without default values?

Reply:
Modified. if no default value, It bypassed the authorization checking

https://github.com/commercetools/commercetools-adyen-integration/pull/584/commits/f8c45d7c1e4cd3e23c8f830d012e43d5fdcbda8a